source/Range.js

```diff
@@ -213,6 +213,7 @@ var insertTreeFragmentIntoRange = function ( range, frag, root ) {
         // Remove trailing <br> – we don't want this considered content to be
         // inserted again later
         cleanupBRs( block, root, false );
+        fixCursor(block, root);
```

Comment:
consider adding a comment why this was necessary

Reply:
changed with 4359e11